examples/resnet/resnet_ptq_cpu.json

```diff
@@ -46,7 +46,14 @@
                     ],
                     "user_config":{
                         "user_script": "user_script.py",
-                        "data_dir": "data",
+                        "data_dir": {
+                            "type": "azureml_datastore",
+                            "config": {
+                                "azureml_client": "azureml_client",
```

Comment:
do we need the user to provide this parameter? For the model path and azureml system, we auto-insert the `azureml_client` if it is not present in the resource config.